open_humans/templates/email/welcome.html

```diff
@@ -15,13 +13,13 @@
 
 <hr>
 
-<b>Add Your Data</b>
+<b>Adding Data</b>
 
 <p>
-  We have many data sources you can add! Each source you add is a generous
-  contribution to science and gets you a badge on your public profile. It also
-  helps enable more studies because researchers can design studies more easily
-  when they know members have data ready to share.
+  Your account can be used to collect and manage any personal data you
+  choose to connect to Open Humans. We have activities created by us, as
+  well as other members, that enable adding data from a variety of sources.
+  enable adding data from a variety of sources.
```

Comment:
@madprime copy paste issue? :)

Reply:
Whoops! Not sure how it happened, I guess I was tired – good catch, thank you!